rebasehelper/plugins/spec_hooks/replace_old_version.py

```diff
@@ -104,7 +116,7 @@ def run(cls, spec_file: SpecFile, rebase_spec_file: SpecFile, **kwargs: Any):
             for index, line in enumerate(section):
                 # special case Version tag to avoid mistakes when version format changes
                 # e.g. in a rebase from 2.5 to 2.5.1, version would be changed to 2.5.1.1
-                if cls._is_local_source(line) or line.startswith('Version'):
+                if cls._is_local_source(line) or any(line.startswith(tag) for tag in cls.IGNORED_TAGS):
```

Comment:
The comment mentions only Version.

Reply:
I moved the comment to the list, I think it is clearer this way